packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -905,7 +921,12 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
       parentElement = getParent(parentElement, ALLOW_VIRTUAL_ELEMENTS);
     }
 
-    return this._root.current;
+    if (parentElement === this._root.current) {
+      return this._root.current;
```

Comment:
Could lines 924-929 just be replaced with `return parentElement`?

Reply:
It could. Done.....